typescript/agentkit/README.md

```diff
@@ -493,6 +511,33 @@ const client = createWalletClient({
 const walletProvider = new ViemWalletProvider(client);
 ```
 
+#### Configuring gas parameters
```

Comment:
Let's add a link to this section in the table of contents https://github.com/coinbase/agentkit/blob/main/typescript/agentkit/README.md#table-of-contents